journals/apps/core/models.py

```diff
@@ -95,11 +95,18 @@ class SiteConfiguration(models.Model):
         help_text=_('JSON string containing OAuth backend settings.'),
         null=False,
         blank=False,
-        default={}
+        default={
```

Comment:
Think we are missing corresponding migration for this change? Don't see it in list of files to commit.